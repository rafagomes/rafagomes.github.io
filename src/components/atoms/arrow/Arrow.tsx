import React from "react";
import styled from "@emotion/styled";

const ArrowDown = styled.span`
    display: block;
    width: 0;
    height: 0;
    margin: 10px auto 10px auto;
    border-left: 10px solid transparent;
    border-right: 10px solid transparent;
    border-top: 10px solid white;
`;

function Arrow() {
    return <ArrowDown />;
}

export default Arrow;
