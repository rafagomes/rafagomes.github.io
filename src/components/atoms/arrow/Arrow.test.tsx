import React from "react";
import { create } from "react-test-renderer";
import Arrow from "./Arrow";

describe("Arrow component", () => {
    it("Arrow component renders as expected", () => {
        const arrow = create(<Arrow />);
        expect(arrow.toJSON()).toMatchSnapshot();
    });
});
