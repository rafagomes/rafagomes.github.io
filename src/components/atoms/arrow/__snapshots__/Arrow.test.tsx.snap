// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arrow component Arrow component renders as expected 1`] = `
<span
  className="css-x8d90d-ArrowDown ektqt6m0"
/>
`;
